README.md

```diff
@@ -374,6 +374,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [OpenWeb Ninja](https://www.openwebninja.com/) - Extremely comprehensive real-time SERP and public data APIs: Google Search, Shopping, Jobs, Images, Lens, News, Google Maps Businesses / Places, Reviews, Photos, Website Emails and Social Contacts Scraper, Amazon, Yelp and more. All APIs include a free tier with 100 to 200 free monthly requests.
   * [Tavily AI](https://tavily.com/) - API for online serach and rapid insights and comprehensive research, with the capability of organization of research results. 1000 request/month for the Free tier with No credit card required.
   * [Zuplo](https://zuplo.com) - Free API Management platform to design, build, and deploy APIs to the Edge. Add Authentication, Rate Limiting, API Documentation, and Monetization in minutes. OpenAPI-native and fully-programmable with web standard apis & Typescript.
+  * [Webhookify](https://webhookify.io/download) - Free desktop app in macos/windows/linux to easily test, debug and integrate webhooks like never before. Generate URLs, Capture Requests, Replay, Repeat. No login/sign up required.
```

Comment:
Sorry this list if for SaaS, only exception really is for generic IDEs

Reply:
Do i need to put on any other section, or my devtool doesn't qualify for this list at all?

Reply:
The list is for SaaS.

Reply:
Got it, thank you